docs/collections_doc.md

```diff
@@ -0,0 +1,108 @@
+## _after_each
```

Comment:
Looks like the support for struct is not there. I guess we'll update later.

Reply:
Fixed!